app/assets/javascripts/leaflet.map.js

```diff
@@ -144,48 +137,88 @@ L.OSM.Map = L.Map.extend({
       name: I18n.t("javascripts.map.base.standard")
     }));
 
-    this.baseLayers.push(new L.OSM.CyclOSM({
+    this.baseLayers.push(new L.MaplibreGL({
       attribution: copyright + ". " + cyclosm + ". " + terms,
       code: "Y",
-      keyid: "cyclosm",
-      name: I18n.t("javascripts.map.base.cyclosm")
+      keyid: "cyclosm", 
+      name: I18n.t("javascripts.map.base.cyclosm"),
+      style: {
+        version: 8,
+        sources: {
+          'cyclosm': {
+            type: 'raster',
+            tiles: [
+              'https://a.tile-cyclosm.openstreetmap.fr/cyclosm/{z}/{x}/{y}.png',
+              'https://b.tile-cyclosm.openstreetmap.fr/cyclosm/{z}/{x}/{y}.png', 
+              'https://c.tile-cyclosm.openstreetmap.fr/cyclosm/{z}/{x}/{y}.png'
+            ],
+            tileSize: 256,
+            attribution: '© OpenStreetMap contributors. Tiles courtesy of OpenStreetMap France',
```

Comment:
Can we keep the attribution localized like in the old code?

```suggestion
            attribution: `${copyright}. ${cyclosm}. ${terms}`,
```